requirements.txt

```diff
@@ -2,7 +2,4 @@ Flask-Cache
 Flask-WTF==0.11
 Flask==1.0
 redis==2.10.5
-hiredis
 requests
-simplejson
-pymongo==2.8.1
```

Comment:
The `pymongo` package provides `bson`: https://pypi.org/project/pymongo/

Please leave `pymongo` in the list of requirements as it's needed.  It should also probably be aligned with the version used in the backend (3.9.0) with a new PR.